blazor/scheduler/appointments.md

```diff
@@ -477,6 +477,63 @@ By default, the scheduler will render the overlapping events based on the start
 ```
 ![Blazor Scheduler with Overlapping Events](images/blazor-scheduler-overlapping-events.png)
 
+## Preventing Overlapping Events
+
+In industries like healthcare and legal services, avoiding overlapping appointments is crucial for compliance and confidentiality. Setting the `AllowOverlap` property to false prevents conflicts and triggers an alert if an overlap is attempted.
```

Comment:
By default, the scheduler renders overlapping events based on their start and end times. To prevent this, set the AllowOverlap property to false. When this property is set, an overlap alert will be displayed during dynamic CRUD actions if new or updated events conflict with existing ones. The conflicting events will be collected in the OverlapCollection within the PopupOpenEventArgs.